### PR TITLE
feat: change email in account settings

### PR DESCRIPTION
**Pull request by JohnXander**

### CODE REVIEW

**Review by sirpae — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Profile from "./components/profile/Profile";
 import EnrolmentPage from "./pages/enrollment";
 import Header from "./components/Header/Header";
 import client from "./utils/client";
+import Account from "./components/account/Account";
```
Comment (Auenc) [inline src/App.js:14]: Let's change these to single quotes to keep inline with the rest of the app

File: src/App.js [hunk 2]
```diff
@@ -53,6 +54,7 @@ function App() {
             <Profile profileData={user} getLoggedInUserId={getLoggedInUserId} user={user} setUser={setUser} />
           } />
         </Route>
+        <Route path="/account" element={<Account user={user} />} />
```
Comment (Auenc) [inline src/App.js:57]: same as above

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ const Header = ({ companyName }) => {
             </Button>
             <Button variant="contained">Add Cohort</Button>
             <Button variant="contained">Logout</Button>
-            <Button href="/profile"><Avatar src="default.png" /></Button>
+            <Button href="/account"><Avatar src="default.png" /></Button>
```
Comment (Auenc) [inline src/components/Header/Header.jsx:57]: Let's change these to single quotes

File: src/components/account/Account.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+function createData(key, value) {
+    return { key, value };
+}
+
+const Account = ({ user }) => {
+    const infoKeys = Object.keys(user)
+    const infoValues = Object.values(user)
+    const rows = [
+        createData(infoKeys[0], infoValues[0]),
```
Comment (Auenc) [inline src/components/account/Account.js:22]: Is there a reason for using a lot of magic numbers here? What are you trying to achieve?

File: src/components/account/Account.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+            <TableContainer component={Paper} className="table-container">
+                <h1>Account Information</h1>
+                <Table sx={{ minWidth: 300, maxWidth: 700 }} aria-label="simple table">
+                    <TableBody>
+                        {rows.map((row) => (
+                            <TableRow
+                                key={row.key}
+                                sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+                            >
+                                <TableCell component="th" scope="row">{row.key}</TableCell>
```
Comment (Auenc) [inline src/components/account/Account.js:43]: Change these quotes to single quotes

File: src/components/account/EditDetails.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+const EditDetails = ({ user }) => {
+    const { email } = user
+    const [open, setOpen] = useState(false)
+
+    const handleClickOpen = () => setOpen(true)
+    const handleClose = () => setOpen(false)
+
+    return (
+        <div>
+            <Button variant="outlined" onClick={handleClickOpen}>Change Email</Button>
```
Comment (Auenc) [inline src/components/account/EditDetails.js:17]: Let's change these to single quotes

File: src/components/account/EditDetails.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+            <Dialog
+                open={open}
+                onClose={handleClose}
+                aria-labelledby="alert-dialog-title"
```
Comment (Auenc) [inline src/components/account/EditDetails.js:21]: Same as above

**Review by Auenc — CHANGES_REQUESTED** [review]
Some minor formatting changes and a question

**Review by Auenc — APPROVED** [review]

**Review by sirpae — APPROVED** [review]

**Pull request merged**